scripts/package

```diff
@@ -15,7 +15,7 @@ cd $(dirname $0)/../package
 
 ../scripts/create-components-file.sh
 
-cp ../bin/rancher.yaml ../bin/rancher-namespace.yaml ../bin/rancher ../bin/agent ../bin/data.json ../bin/k3s-airgap-images.tar .
```

Comment:
`../bin/rancher.yaml` and  `../bin/rancher-namespace.yaml` are generated by the `build-rancherd` script.